openslides_backend/action/actions/poll/update.py

```diff
@@ -96,3 +103,20 @@ def check_100_percent_base(self, instance: Dict[str, Any]) -> None:
             )
             pollmethod = poll.get("pollmethod")
         base_check_100_percent_base(pollmethod, onehundred_percent_base)
+
+    def check_state_change(
+        self, instance: Dict[str, Any], poll: Dict[str, Any]
+    ) -> bool:
+        if poll.get("type") != Poll.TYPE_ANALOG:
+            return False
+        if poll.get("state") != Poll.STATE_CREATED:
```

Comment:
This check has to be removed (analog to poll.create)

Reply:
see above.